### DISCUSSION TITLE
Windows on Arm

### DISCUSSION THREAD

**Discussion by Confusingboat** [ngrok/ngrok#47] [Ideas]
With the growing popularity of Windows on Arm (WoA) it would be nice to see an arm64 build for Windows.

**Comment by russorat** [comment]
@Confusingboat thanks! let me ask the team how hard it would be to add that to our build process.

**Comment by Abdiramen** [comment]
Hey @Confusingboat, I just built an unstable release of ngrok for WoA. If you have a chance could you check it out?

https://dl.equinox.io/ngrok/ngrok-v3/unstable

**Comment by Confusingboat** [comment]
Wow that was a quick turnaround! I'll test this out as soon as I'm able 🙂

**Reply by Abdiramen** [reply]
Hey @Confusingboat ! Have you had a chance to try out the ngrok WoA release? I'm looking forward to your feedback.

**Reply by Confusingboat** [reply]
@Abdiramen I gave it a quick test (configured and served a web page from a local container) and it seems to work just fine 🙂

Also can confirm it is running native arm64
<img width="415" alt="image" src="https://github.com/user-attachments/assets/784cec50-b549-4357-afc6-be93c636b403">

**Reply by Abdiramen** [reply]
Awesome! I'm glad it's working for you, and I hope you have of fun with it!
